docs/api.md

```diff
@@ -201,6 +201,64 @@ Example Request Body:
 Returns:
 `201` if successful
 
+
+### POST /v1/dex/orders/replace
+
+Description:
+Replace an existing order in the orderbook.
+
+Please Note:
+
+* There is a limit of 50 active orders on each book per-side. If you exceed this limit,
+your request will return `400` and will not be added to the book.
+
+* Your request will return `201`, but the order itself will still have a status of `PENDING` until
+it is processed by our internal matching engine.
```

Comment:
"the new order itself"

Also mention that the canceled order may still not be canceled until processed by the internal matching engine